spec/controllers/parcels_controller_spec.rb

```diff
@@ -81,6 +80,12 @@
       expect(assigns(:parcel)).to be_a_new(Parcel)
     end
 
+    it "assigns blank SenderInfo and RecipientInfo to @parcel" do
+      get :new
+      expect(assigns(:parcel).sender_info).to be_a_new(SenderInfo)
+      expect(assigns(:parcel).recipient_info).to be_a_new(RecipientInfo)
```

Comment:
Coś ostatnio wspominałeś o `assigns`'ach? Nie jestem przekonany, czy to jest bieżąco zalecana składnia, nie miałem z tym styku za dużo.